Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM python:3.7-buster as base
-RUN pip install poetry
+FROM python:3.10-bullseye as base
+RUN pip install poetry==1.1.14
 COPY poetry.toml poetry.lock pyproject.toml /
 
 FROM base as production
-RUN poetry install -n --without dev
+RUN poetry install -n
```

Comment:
Is there a reason you've told your production container to now install dev dependencies? `--no-dev` should work for this version of poetry